index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PORTFOLIO</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  </head>
+  <body>
+
+        <section class= "recent-works">
+          <h1>My Recent Works</h1>
```

Comment:
[]OPTIONAL] Well done👍, Kindly center this headline to be inline with the template.☺
**Your Design**
![cent](https://user-images.githubusercontent.com/73854999/198846953-07c0138b-23e9-47fd-8035-ece6c671c918.PNG)

**Figma Design**
![center](https://user-images.githubusercontent.com/73854999/198846961-eed07a6c-f9e1-4ca0-8ee8-773db8ae57e4.PNG)